src/app/pages/send/send-crypto-asset-form/form/stacks/stacks-common-send-form.tsx

```diff
@@ -75,6 +76,8 @@ export function StacksCommonSendForm({
                 <FormFooter balance={availableTokenBalance} />
                 <HighFeeDrawer learnMoreUrl={HIGH_FEE_WARNING_LEARN_MORE_URL_STX} />
                 <Outlet />
+                {/* This is for testing purposes only, to make sure the form is ready to be submitted */}
+                {fees ? <Box data-testid={SendCryptoAssetSelectors.SendPageReady}></Box> : null}
```

Comment:
Maybe we can just use a `<span` for stuff like this that's not really visible

Reply:
Yep, I'll change it in the btc form too.